src/constants/chainInfo.ts

```diff
@@ -223,6 +224,31 @@ const CHAIN_INFO: ChainInfoMap = {
     color: '#35D07F',
     backgroundColor: '#34d07f1f',
   },
+  [SupportedChainId.AURORA]: {
+    networkType: NetworkType.L1,
+    blockWaitMsBeforeWarning: ms`10m`,
+    bridge: 'https://rainbowbridge.app/transfer',
+    docs: 'https://aurora.dev/',
+    explorer: 'https://explorer.aurora.dev/',
+    infoLink: 'https://info.uniswap.org/#/',
```

Comment:
Not sure about this but i dont know what to put

Reply:
Since it is a fork of uniswap widget it would contain info about uniswap, but this is of no use to us.
We can ignore and leave it blank.